web/api/v1/api.go

```diff
@@ -344,16 +351,20 @@ type Target struct {
 	// Any labels that are added to this target and its metrics.
 	Labels model.LabelSet `json:"labels"`
 
-	ScrapeUrl string `json:"scrapeUrl"`
+	ScrapeURL string `json:"scrapeURL"`
```

Comment:
I think in general in JSON it would stay `scrapeUrl`, no? 
Are we doing the full capsing elsewhere?